tools/cxbe/Xbe.cpp

```diff
@@ -1071,7 +1072,7 @@ void Xbe::DumpInformation(FILE *x_file)
     fprintf(x_file, "Size of Headers                  : 0x%.08X\n", m_Header.dwSizeofHeaders);
     fprintf(x_file, "Size of Image                    : 0x%.08X\n", m_Header.dwSizeofImage);
     fprintf(x_file, "Size of Image Header             : 0x%.08X\n", m_Header.dwSizeofImageHeader);
-    fprintf(x_file, "TimeDate Stamp                   : 0x%.08X (%s)\n", m_Header.dwTimeDate, BetterTime(ctime((const long*)&m_Header.dwTimeDate)));
+    fprintf(x_file, "TimeDate Stamp                   : 0x%.08X (%s)\n", m_Header.dwTimeDate, BetterTime(ctime((const time_t*)&m_Header.dwTimeDate)));
```

Comment:
Hmm.. Is this right? Can we be sure that type `time_t` is a 32-bit value?

Reply:
Seeing as `time_t` is undefined in the C standard according to [cppreference.com](https://en.cppreference.com/w/c/chrono/time_t), you're probably right that we can't. `clang` would not compile with just a regular `long int*` though.
Suggestions welcome!

Reply:
After a quick discussion on Discord, @mborgerson suggested that a conversion of the `long int` was done before sending a pointer to `ctime()`, instead of converting the pointer.